docs/changelog/92879.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 92879
+summary: Rename GET /_health to GET /_health_report
```

Comment:
As _health wasn't released, should this PR talk about the GAing of the health API? 
e.g. a release highlight as was initially written https://github.com/elastic/elasticsearch/pull/92420/files#diff-c64592fdfcabda04efcbe96384e6cc999f1488efd984c6dde5f4567526b4bcd4
(I think docs/changelog/92420.yaml should be deleted)

And maybe this PR should have the release highlights label as opposed to #92420 ?